### PR TITLE
OH2-388 | Exam - implement multiple options procedure

### PR DESCRIPTION
**Pull request by SteveGT96**
See [OH2-388](https://openhospital.atlassian.net/browse/OH2-388)

Paired with [#483](https://github.com/informatici/openhospital-api/pull/483)

[OH2-388]: https://openhospital.atlassian.net/browse/OH2-388?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/generated/runtime.ts [hunk 1]
```diff
@@ -1,266 +1,211 @@
...
 
-import { Observable, of, Subscriber } from "rxjs";
-import { ajax, AjaxRequest, AjaxResponse } from "rxjs/ajax";
-import { map, concatMap } from "rxjs/operators";
+import { Observable, of, Subscriber } from 'rxjs';
+import { ajax, AjaxRequest, AjaxResponse } from 'rxjs/ajax';
+import { map, concatMap } from 'rxjs/operators';
 
-export const BASE_PATH = "https://oh2.open-hospital.org".replace(/\/+$/, "");
+export const BASE_PATH = 'http://localhost:8080'.replace(/\/+$/, '');
```
Comment (mwithi) [inline src/generated/runtime.ts]: is this change to be reverted?

Comment (SteveGT96) [reply src/generated/runtime.ts]: No ! This is a generated code we shouldn't revert it. Don't worry about that. It's possible to change base path using an env variable(const basePath = process.env.REACT_APP_BASE_PATH || BASE_PATH).

**Comment by mwithi** [comment]
Please resolve the conflicts

**Review by SilverD3 — APPROVED** [review]
LGTM!

**Review by mwithi — APPROVED** [review]
Reviewed and approved by @SilverD3

**Pull request merged**